articles/vmware/vmw-faq.md

```diff
@@ -552,13 +541,15 @@ The recovery time objective (RTO) depends on the nature of the disaster or failu
 
 If you need more control and assurance around how data is replicated, the DR service is tested, or the solution handles failover and failback, we recommend you create your own DR solution by using independent sets of VMs in the various regions offered by the UKCloud platform.
 
+An alternative UKCloud solution to acheive DR with a low RPO and RTO is [Journaling Protection](vmw-sco-journaling-protection.md).
+
 ### How does UKCloud enable customers to create their own DR solutions?
 
-Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or in the application.
+Although we have engineered our cloud platform to tolerate failures and ensure customers services remain available, we encourage all our customers to design for failure and build DR capabilities into the system design, or into the application.
 
 To help this process, the UKCloud platform has been designed to give you the options you need to design a disaster tolerant solution. UKCloud's platform is about providing customers with choice. Clearly, customers need to balance the low probability of these failure scenarios occurring with the cost and complexity involved in mitigating their impact.
 
-Depending on the type of failure you are trying to mitigate, UKCloud has been developed to allow you design appropriate resilience into your solution.
+Depending on the type of failure you are trying to mitigate, UKCloud services have been developed to allow you design appropriate resilience into your solution.
```

Comment:
Issue existed in the original text but while you're editing, could you change
developed to allow you design
to
developed to enable you to design

(change allow to enable and add the missing to)